install.sh

```diff
@@ -6,9 +6,13 @@
 ### Ubuntu 14.*
 ### Ubuntu 15.04
 
-echo "Check Ubuntu Kernel"
-
 ## Check ubuntu kernel to correcty 
+## kernel_version recebe uma linha do filtro do "uname -r | sed 's/\(\-[0-9]* ..." para saber
+## se o kernel é compativel com a instalação da PS4eye
+
+## se kernel_version for menor que kernel_reference, mostra uma mensagem de error e sai da instalação
```

Comment:
OK!! Documentação em português está boa, porém tem de explicar um pouco mais em inglês. 
Se tiver duvidas pergunta a Kai, ele ajuda.

Melhore a organização dos comentários. Quando for em inglês coloque uma indicação de qual lingua no inicio, ex:
## EN -> inglês
## PT -> português.